crates/swc_plugin_runner/Cargo.toml

```diff
@@ -22,11 +22,9 @@ plugin_transform_host_js = [
 plugin_transform_host_native = [
   "wasmer/default",
   "wasmer-wasix/sys",
```

Comment:
I removed all default enabled feature for wasmer-wasix/sys

Reply:
Is it possible to upstream something to wasmer to make this configurable without needing a fork?

Reply:
yeah, that's the plan

Reply:
https://github.com/wasmerio/wasmer/issues/5612 feature request for wasmer

Reply:
@bgw @kdy1  turns out it's not a trivial feature for wamser-wasix to implement subset of wasi features, so there're some options here
* waiting until wasmer team  implement subset of wasi features
* temporarily fork and use in swc and when wasmer support subset of wasi then change to wamer version
* consider move to wasmtime + wasi-common(like zed did), it's 14M vs 23M(wasmer + wasmer-wasix)

Reply:
I think we should try wasmtime, but I'm not sure about the performance.

Reply:
I can give wasmtime a try and compare the performance,considering they both use Cranelift as backend so maybe no big performance expected

Reply:
cc @bgw Any thoughts?